lib/chef/resource/snap_package.rb

```diff
@@ -54,9 +63,7 @@ class SnapPackage < Chef::Resource::Package
       allowed_actions :install, :upgrade, :remove, :purge
 
       property :channel, String,
-        description: "The default channel. For example: stable.",
-        default: "stable",
-        equal_to: %w{edge beta candidate stable},
+        description: "The desired channel. For example: `latest/stable` or `0.x/edge`. If nil, the resource will install the snap's default version.",
```

Comment:
Leave the `default:` such that it resolves to `latest/stable` in the logic, and then allow an explicit `nil` for the default version?